Cargo.toml

```diff
@@ -74,6 +75,10 @@ auto-initialize = []
 # Optimizes PyObject to Vec conversion and so on.
 nightly = []
 
+# Provides a (lossy) From<anyhow::Error> impl for PyErr.
+# There is currently no way to convert the PyErr back into the original anyhow::Error.
+anyhow-integration = ["anyhow"]
```

Comment:
There's no need for this to have a separate feature - users can just enable `anyhow` feature directly.

Reply:
Didn't know that, will fix!